prism-agent/service/server/src/main/scala/io/iohk/atala/iam/authorization/keycloak/admin/KeycloakPermissionManagementService.scala

```diff
@@ -0,0 +1,16 @@
+package io.iohk.atala.iam.authorization.keycloak.admin
+
+import io.iohk.atala.iam.authorization.core.PermissionManagement
+import io.iohk.atala.shared.models.WalletId
+import org.keycloak.admin.client.Keycloak
+import zio.IO
+
+import java.util.UUID
+
+class KeycloakPermissionManagementService(keycloak: Keycloak) extends PermissionManagement.Service {
```

Comment:
we might need keycloak `authz` client here instead of admin client.

Reply:
I'm also thinking about this. Let's use the `admin` client. I will refactor it later.